src/transformers/modeling_utils.py

```diff
@@ -2480,7 +2502,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                         "proxies": proxies,
                         "resume_download": resume_download,
                         "local_files_only": local_files_only,
-                        "use_auth_token": use_auth_token,
+                        "use_auth_token": token,
```

Comment:
some methods still use `use_auth_token` and will be changed only in follow up PR.